traveler-swift-core/TravelerKit/Models/Product/Purchased/PurchasedBookingProduct.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  PurchasedBookingProduct.swift
+//  TravelerKit
+//
+//  Created by Rakin Hoque on 2019-11-28.
+//  Copyright © 2019 Ata Namvari. All rights reserved.
+//
+
+import Foundation
+
+/// Any purchased booking product
+public struct PurchasedBookingProduct: PurchasedProduct, Decodable {
+    /// Identifier
+    public let id: String
+    /// Order identifier
+    public let orderId: String
+    /// Order reference number
+    public let orderReferenceNumber: String?
```

Comment:
If its a reference number it should be modeled as an Int right? otherwise is an orderReferenceIdentifier.

Reply:
The `Order` model also defines it as referenceNumber: String?